charts/opensearch-operator/templates/OpensearchUserRoleBinding-crd.yaml

```diff
@@ -58,7 +62,6 @@ spec:
             required:
             - opensearchCluster
             - roles
-            - users
```

Comment:
Didn't understand why to delete the users?

Reply:
Hello! We can add only users or only backendRoles to mapping. So I delete users from requered fields. Maybe we can implement choice between backendRoles and users here... But I don't know how to do this.

I did some manual tests without users and backendRoles fields in OpensearchUserRoleBinding manifest like this:
```yaml
apiVersion: opensearch.opster.io/v1
kind: OpensearchUserRoleBinding
metadata:
  name: empty-rolebinding
spec:
  opensearchCluster:
    name: opensearch-cluster
  roles:
  - test-role

```
and didn't get any errors.

Reply:
IMO a combination of users and backendRoles is possible. And if you tested and leaving out both users and backendRoles doesn't produce errors then it's ok for me.